apache2/Makefile.am

```diff
@@ -6,6 +6,7 @@ mod_security2_la_SOURCES = acmp.c \
     ag_mdb/ag_mdb.cpp \
     waf_logging/waf_format.pb.cc \
     waf_logging/waf_log_util.cc \
+    waf_lock/waf_lock.cpp \
```

Comment:
Can we separate the commit? One is for your standalone persistent lock utility code, another is for the ModSec-integration code (WAF JSON Log may be the independent one), and the other is for configuration and Makefile.

Reply:
yes